ncm-ntpd/src/main/pan/components/ntpd/config.pan

```diff
@@ -5,5 +5,14 @@
 
 unique template components/${project.artifactId}/config;
 
-include { 'components/${project.artifactId}/config-common' };
-include { 'components/${project.artifactId}/config-rpm' };
+include 'components/${project.artifactId}/schema';
+
+bind "/software/components/${project.artifactId}" = component_ntpd_type;
+
+"/software/packages" = pkg_repl("ncm-${project.artifactId}", "${no-snapshot-version}-${rpm.release}", "noarch");
+
+prefix '/software/components/${project.artifactId}';
+'active' ?= true;
+'dispatch' ?= true;
+'version' ?= '${no-snapshot-version}';
```

Comment:
What's the purpose of this attribute? I thought it was to trigger a re-run of the component after the version changed but I recently discovered that ncm-cdispd watches /software/packages/ncm-$component and re-runs any components that has been upgraded. Is this still useful or should be it be dropped?